core/src/main/scala/org/broadinstitute/dsde/rawls/entities/base/AuditLoggingEntityProvider.scala

```diff
@@ -83,7 +83,7 @@ class AuditLoggingEntityProvider(val delegate: EntityProvider,
     )
 
     import AuditJsonSupport._
-    logger.info("Entity operation audit", StructuredArguments.raw("audit", auditInfo.toJson.compactPrint))
+    logger.trace("Entity operation audit", StructuredArguments.raw("audit", auditInfo.toJson.compactPrint))
```

Comment:
this just reduces log volume for the re-migration. Since this is all happening in a new instance with a new database we don't need the audit logging.

Reply:
Just checking - we have other logging that will give us insight into what is happening with the migrations?

Reply:
I've improved the logging in the most recent commits to (try to) make sure we have enough insight!